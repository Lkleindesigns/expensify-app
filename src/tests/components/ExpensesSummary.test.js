import React from 'react'
import { shallow } from 'enzyme'
import numeral from 'numeral'
import { ExpensesSummary } from '../../components/ExpensesSummary'

test('Should correctly render expenses total with 1 expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={12} />)
    expect(wrapper).toMatchSnapshot()
})

test('Should correctly render expenses total with multiple expenses', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={23} expensesTotal={12345} />)
    expect(wrapper).toMatchSnapshot()    
})