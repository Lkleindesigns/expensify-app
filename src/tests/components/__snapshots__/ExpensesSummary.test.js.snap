// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should correctly render expenses total with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $0.12
  </h1>
</div>
`;

exports[`Should correctly render expenses total with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     totalling 
    $123.45
  </h1>
</div>
`;
